src/utils/transaction.ts

```diff
@@ -671,39 +560,61 @@ export const buildCrossChainAction = async (
               decimals: fromAssetDecimals,
               logoURI: fromAssetIconUrl,
             },
+            toAsset: { decimals: toAssetDecimals },
             amount,
-            offer,
```

Comment:
no longer doing swaps?

Reply:
In the Staking flow we don't need offers, so removed that.